db/structure.sql

```diff
@@ -836,6 +836,35 @@ CREATE SEQUENCE public.certificates_oid_seq
 ALTER SEQUENCE public.certificates_oid_seq OWNED BY public.certificates_oplog.oid;
 
 
+--
+-- Name: changes; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.changes (
+    id integer NOT NULL,
+    goods_nomenclature_item_id text,
+    goods_nomenclature_sid text,
```

Comment:
Do we want to update the type of this one?

Reply:
True, I had forgot this, sorry